File: Client/Pong/src/models/player.ts
export class Player {
    public connectionId: string;
    public nickname: string;
    public isChallengingMe: boolean = false;
    public isChallengedByMe: boolean = false;
    public isInGame: boolean = false;

    constructor(_connectionId: string, _nickname: string)
    {
      this.connectionId = _connectionId;
      this.nickname = _nickname;;    }
}

File: Server/PongMultiplayer/Models/Player.cs
namespace PongMultiplayer
{
    public class Player
    {
        public string ConnectionId { get; set; }
        public string Nickname { get; set; }
        public bool IsInGame { get; set; } = false;
        public string OpponentConnectionId { get; set; }
        public double PaddlePositionY { get; set; }
    }
}
